server/session.py

```diff
@@ -587,3 +588,144 @@ def notify(self, height, touched):
             self.mns.add(vin)
             return result.get(vin)
         return None
+
+class PacElectrumX(ElectrumX):
```

Comment:
Please extend the DashElectrumX and override any methods. Even better, extend DashElectrumX if your additions apply to Dash other master-node coins as well.